src/extensions/default/bramble/lib/RemoteCommandHandler.js

```diff
@@ -233,6 +224,19 @@ define(function (require, exports, module) {
         case "BRAMBLE_PROJECT_SIZE_CHANGE":
             UI.setProjectSizeInfo(args[0]);
             break;
+        // CDO-Bramble: The remaining commands are custom to our fork.
```

Comment:
again, nice! keeping all of our custom changes grouped together will make updates much easier I think!